CommonTools/ParticleFlow/plugins/PFnoPileUpPacked.cc

```diff
@@ -0,0 +1,95 @@
+#ifndef CommonTools_ParticleFlow_PFnoPileUpPacked_
```

Comment:
I assume noboby will ever include this plugin: therefore this `#ifndef` -> `#define` -> `#endif` structure is not needed